kernel/boot_info/src/lib.rs

```diff
@@ -95,8 +95,8 @@ pub enum Address {
 impl fmt::Debug for Address {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Address::Physical(paddr) => write!(f, "p{:#X}", paddr),
-            Address::Virtual(vaddr)  => write!(f, "v{:#X}", vaddr),
+            Address::Physical(paddr) => write!(f, "p{paddr:#X}"),
+            Address::Virtual(vaddr)  => write!(f, "v{vaddr:#X}"),
```

Comment:
please merge in the latest changes from main, this was removed yesterday or the day before